index.js

```diff
@@ -774,6 +774,18 @@ class Encore {
         return this;
     }
 
+    /**
+     * Call this if you wish to disable the CSS
+     * url handler.
+     *
+     * @returns {Encore}
+     */
+    disableUrlHandling() {
+        webpackConfig.disableUrlHandling();
+
+        return this;
```

Comment:
Hmm, I think a better idea would be to allow the `css-loader` to be configured in general. So, to create a method more like `configureBabel()`, where you pass it a callback, and we pass the callback the final config (which you can then modify). That'll allow you to disable the url handling... but also to modify any options on the `css-loader`.